doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+SYSERR(dir) | MALE | pgm dir | Define directory for output ERR files. This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSLKD(file) | L | pgm dir | Define input source file for linker commands such as INCLUDE, ALIAS, ENTRY, and NAME.
+SYSLOG(dir) | E | pgm dir | Define directory for output LOG file. This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSLST(dir) | AL | pgm dir | Define directory for output LST files.  This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSMAC(dir) | M | mac dir | Define one or more directories for source MAC files.  If the option starts with + the directories listed will be appended to the maclib list.  Multiple directories can be specified in one parameter and are always separated by +. When you specify this option more than once and want previous parameter instructions to remain in place, ensure that you start parameter with a `+` otherwise it will drop prior directories.  This option may also override suffix by adding *.sfx.
+SYSMLC(dir) | MA | pgm dir | Define directory for source input MLC files.
+SYSOBJ(dir) | AL | pgm dir | Define directory for OBJ relocatible object files
+SYSOPT(dir) | MALE | pgm dir | Define directory for @file option files.  This option may override suffix using *.sfx.
+SYSPARM('text') | M | none | Define text string which can be accessed by mz390 global macro variable &SYSPARM.
+SYSPCH(dir) | M | pgm dir | Define directory for PCH output files from PUNCH.  This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).  Note PUNCH output is in ASCII source format and is not directed to the OBJ file.  These files may be used as linker input commands via SYSLKD(file) option.  Set to SYSPCH(.) for current directory when no path is specified on DSNAME parm for PUNCH.
+SYSPRN(dir) | A | pgm dir | Define directory for assembler listing PRN output files.  This option may also override file name and suffix.  This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSTERM(file) | MALE | pgm.ERR | Define alternate file for all start/end messages plus any error messages, plus related source file statistics for each file containing errors.  The default is the program name with ERR suffix.
+SYSTRC(dir) | MALE | pgm dir | Define directory for output TRACE TR? files
+TEST | E | NO | Start execution with interactive TEST mode active which prompts for commands such as T opcode or G opcode to trace or execution to the next occurrence of opcode name.
+TEST(ddname) | E | NO | Define input command file to run TEST commands.  `ddname` is an environment variable set to the name of the command file. See regression test rt\test\TESTTST1.MLC for example.
```

Comment:
Line 105.
1. I do not see "TEST(ddname)" in the z390.org doc.
2. Why does "ddname" appear red and inside a box when viewed at adelosa.github.io?
3. The z390.org doc file has a "TESTDD(file)" at this table position. It is not here.

Reply:
1. There was issue #109 which said that TESTDD was wrong, should have been TEST.. Refer to the issue for details.
2. The red letter is just formatting applied.. Will be removed for consistence.
3. See item 1

Reply:
Item 2 - Fixed 40c73aa